### ISSUE TITLE
Document HalfSpace and HyperPlane

### ISSUE THREAD

**Issue by mforets** [JuliaPolyhedra/Polyhedra.jl#68]
`HalfSpace` and `HyperPlane`, as well as other related types, are defined in `src/elements.jl`.

**Comment by blegat** [comment]
I am thinking of adding a section before Representation to define each element. Then the section on representation will use them as building blocks like in https://github.com/JuliaPolyhedra/Polyhedra.jl/issues/41.
But also things like
```julia
HyperPlane([1, 1], 1) ∩ HyperPlane([-1, 0], 0)
```
which would be equivalent to
```julia
hrep([HyperPlane([1, 1], 1), HyperPlane([-1, 0], 0)])
```

**Comment by blegat** [comment]
Fixed by 735a272fa79cc05bdf5c341f4f548716507a98d2
